{"name": "Проверка ручки логина пользователя с неверным логином", "status": "passed", "description": "Сравниваем ожидаемый статус-код ответа \"401\" с фактическим", "start": 1701645203849, "stop": 1701645203983, "uuid": "f4078187-02e9-497f-b6f1-14dd5ac544e1", "historyId": "5ccfc0707696cb8719c14a56db4585a0", "testCaseId": "5ccfc0707696cb8719c14a56db4585a0", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_false_login_check", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "WIN-48K729GVFIE"}, {"name": "thread", "value": "13352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}